cm-frontend/src/styles/_register.scss

```diff
@@ -0,0 +1,46 @@
+.register {
+    max-width: 800px;
+    margin: 20vh auto;
+    padding: 10%;
+    text-align: center;
+    position: relative;
+    background-color: $light-purple-transparent-background;
+    
```

Comment:
since we're using sass, you should put register class children INSIDE .register, that means, you should put label inside .register tag, not '.register label'